source/doi_howto.rst

```diff
@@ -0,0 +1,124 @@
+How-to
```

Comment:
Most of this how-to is not DOI specific, could that be moved to a "howto add metadata" section?

Reply:
I agree with you, although I think it would be good to have a list of steps from beginning  to end. Documentation of data is a key step to have MET provide consistently good products. But this apply also for data without a DOI. I could make another section and link the DOI documentation to it. We can reshuffle when the documentation is more complete.

Reply:
Ok, I agree the howto should be kept in one piece. We can reshuffle when it's more clear what the registration process should be in general, independent of whether the data producer wants DOI or not.

Reply:
Ok. We leave it as it is then and change later on.